RecoHI/Configuration/python/customise_PPwithHI.py

```diff
@@ -136,12 +136,47 @@ def storeCaloTowersAOD(process):
         process.AODSIMoutput.outputCommands.extend(['keep *_towerMaker_*_*'])
 
     return process
+	
+# Add rhoProducer to AOD event content
+def addRhoProducer(process):
+    process.load('RecoJets.JetProducers.kt4PFJets_cfi')
+    process.load('RecoHI.HiJetAlgos.hiFJRhoProducer')
+    process.load('RecoHI.HiJetAlgos.hiFJGridEmptyAreaCalculator_cff')
+
+    process.kt4PFJets.doAreaFastjet = True
+    process.kt4PFJets.jetPtMin      = cms.double(0.0)
+    process.kt4PFJets.GhostArea     = cms.double(0.005)
+    process.hiFJGridEmptyAreaCalculator.doCentrality = False
+    
+	# extend AOD content
+    process.reconstruction *= process.kt4PFJets
+    process.reconstruction *= process.hiFJRhoProducer
+    process.reconstruction *= process.hiFJGridEmptyAreaCalculator
+
+    # extend AOD content
+    if hasattr(process,'AODoutput'):
```

Comment:
ah - ok, so they could go into the normal pp event contents. but maybe that causes a problem..